package.json

```diff
@@ -63,20 +63,20 @@
     "@sphereon/ssi-sdk-ext.key-manager": "0.24.1-unstable.136",
     "@sphereon/ssi-sdk-ext.key-utils": "0.24.1-unstable.136",
     "@sphereon/ssi-sdk-ext.kms-musap-rn": "0.24.1-unstable.136",
-    "@sphereon/ssi-sdk.agent-config": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.contact-manager": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.core": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.credential-store": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.data-store": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.event-logger": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.issuance-branding": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.mdl-mdoc": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.oid4vci-holder": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.sd-jwt": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.siopv2-oid4vp-op-auth": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.vc-handler-ld-local": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.xstate-machine-persistence": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-types": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
+    "@sphereon/ssi-sdk.agent-config": "0.30.2-feature.SPRIND.92.149",
```

Comment:
Don't forget to updated after https://github.com/Sphereon-Opensource/SSI-SDK/pull/259 is merged